features/support/env.rb

```diff
@@ -74,8 +74,7 @@ def debrand_string(str)
 end
 host = "https://#{host}"
```

Comment:
Umm, converting a hostname to a url and calling it `host`. Violates the principle of least surprise.